src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/resources/stylesheet.css

```diff
@@ -910,10 +956,10 @@ div.page-search-info:hover button#page-search-copy span {
     opacity: 90%;
 }
 div.page-search-info button#page-search-copy:hover {
-    background-color: #dfe6f1;
+    background-color: rgba(128, 128, 160, 0.2);
```

Comment:
why have the old hex values just been converted to `rgba` values, and not CSS custom properties?

Reply:
One of my intentions was to keep the number of CSS custom properties as small as possible. I think the hover color of the copy-to-clipboard button (in this case for the search page extra information) is "fringe" enough to not want to define a dedicated property for it.

Using a `rgba` value with a middle-of-the-road blue-grey color and a low alpha value allowed generate a semitransparent overlay that works both with light and dark background colors. The only background color this would not work for is if the background was itself a medium grey color, but in this case it could be overridden with a extra CSS rule.

Reply:
mmmm ... OK, but ...

_Keep everything as simple as possible, but no simpler_

Yes, there is merit it reducing the number of CSS custom properties, but there is equal merit in a simple rule that says that all colors are set via custom properties.   As it stands, we have to explain the (reason for the) exception,